### DISCUSSION TITLE
What's the url of the RUNNING tab

### DISCUSSION THREAD

**Discussion by yanghaogn** [Netflix/conductor#2799] [General]
I have try to use conductor at  [https://netflix.github.io/conductor/labs/beginner/](https://netflix.github.io/conductor/labs/beginner/)

What's the url to see the the RUNNING tab 
![image](https://user-images.githubusercontent.com/5836768/156131626-f2e9363b-a22c-438b-b112-0a174b854f98.png)

**Comment by yanghaogn** [comment]
I have not see the tab at [http://localhost:5000/](http://localhost:5000/)

**Comment by yanghaogn** [comment]
I have found the answer in [https://github.com/Netflix/conductor/discussions/2780](https://github.com/Netflix/conductor/discussions/2780)

As [http://localhost:5000/execution/<workflow_id>](http://localhost:5000/execution/<workflow_id>)

**Reply by apanicker-nflx** [reply]
The above screenshot is from the old UI, new UI does not have a RUNNING tab. You can search by RUNNING status and click on the execution in the search result to navigate to the workflow execution page.

**Reply by yanghaogn** [reply]
Thanks. And when server is in memory without ES, the search function is not ready, we can visit by [http://localhost:5000/execution/<workflow_id>](http://localhost:5000/execution/%3Cworkflow_id%3E)
